.travis.yml

```diff
@@ -28,7 +28,7 @@ env:
         - MAIN_CMD='python setup.py'
         - CONDA_DEPENDENCIES='Cython jinja2'
         - CONDA_ALL_DEPENDENCIES='Cython jinja2 scipy h5py matplotlib pyyaml scikit-image pandas pytz beautifulsoup4 ipython mpmath bleach'
-        - PIP_DEPENDENCIES=''
+        - PIP_DEPENDENCIES='pytest-astropy'
```

Comment:
can we add the ``pytest-astropy`` dependency directly to setup.py?

Reply:
It's already there but for some reason it didn't work without being explicit here.

Reply:
Astropy is not installed before running the tests, hence why ``install_requires`` is not used I think.

Reply:
Yes that's true, although the error is actually a segfault that occurs while building cython extensions. This seems pretty weird to me, but I can reproduce it locally, and will try to take a closer look tomorrow.